libtenzir/include/tenzir/plugin.hpp

```diff
@@ -461,6 +461,16 @@ class plugin_parser {
     -> std::optional<generator<table_slice>>
     = 0;
 
+  /// Apply the parser to an array of strings.
+  ///
+  /// The default implementation of creates a new parser with `instantiate()`
+  /// for every single string.
+  ///
+  /// @post `input->length() == result_array->length()`
+  auto parse_strings(std::shared_ptr<arrow::StringArray> input,
+                     operator_control_plane& ctrl) const
+    -> std::vector<std::pair<type, std::shared_ptr<arrow::Array>>>;
```

Comment:
Is this non-`virtual` on purpose?

Reply:
Seems like I committed the fix to this in my key-value parser branch instead of this one. Fixed! :+1: